### ISSUE TITLE
Question about discrepancy between released checkpoint evaluation and reported 3D metrics

### ISSUE THREAD

**Issue by losehu** [mshooter/DigiDogs_release#5]
Hi, thank you very much for making this repository publicly available.

I have been trying to reproduce the reported results, and I would really appreciate your help with two questions regarding evaluation and training reproducibility.

First, I evaluated the released checkpoint using:

python scripts/launch.py test --config digidogs/configs/test_digidogs.yaml --ckpt_path checkpoints/gta_poseF16/epoch=15_val_loss=0.4664.ckpt

The results I obtained are:

mpjpe2d: 0.056207145233445666
mpjpe3d: 0.28341376781463623
pa mpjpe3d: 0.17432700097560883
pa pck3d: 0.7038256525993347
pck2d: 0.9529311656951904
pck3d: 0.5161510109901428

The 2D metrics seem to be quite close to those reported in the paper, while the 3D metrics appear to be lower than expected. I was wondering if I might be misunderstanding the evaluation protocol.

Could you kindly clarify the following points?

- Which logged metrics correspond to the 3D results reported in the paper?
- Should the paper’s 3D numbers be compared with `pa mpjpe3d` / `pa pck3d`, or with `mpjpe3d` / `pck3d`?
- Is there any difference between the released evaluation code and the exact evaluation setting used for the paper (for example, visible-joint masking, normalization, Procrustes alignment, or the test split)?

Second, I also tried training the model on an RTX 4090 using the default repository settings.

During training, I commented out the code in ENABLE TEST related to:

torch.sum(v).item() != 26

With this setup, the validation loss stopped decreasing at around epoch 18. After that, I evaluated the trained model with `python scripts/launch.py test`, and obtained the following results:

mpjpe2d: 0.0590914843284582
mpjpe3d: 0.32569459080696106
pa mpjpe3d: 0.21974527835845947
pa pck3d: 0.5702748894691467
pck2d: 0.9513282179832458
pck3d: 0.4185384511947632

These results seem to be worse than both the released checkpoint and the paper results.

So I would also be very grateful if you could clarify:

- Whether the weights in `checkpoints/` are the same ones used for the final evaluation in the paper
- Whether my testing procedure is correct
- Whether there are any additional training details that are important for reproducing the reported 3D results, such as data filtering, masking, early stopping settings, evaluation code version, or sensitivity to random seeds

Thank you very much for your time and help. Any guidance would be greatly appreciated.
